arangod/Graph/Graph.cpp

```diff
@@ -417,9 +466,10 @@ void EdgeDefinition::toVelocyPack(VPackBuilder& builder) const {
     builder.add(VPackValue(to));
   }
   builder.close();  // array
+  builder.add("type", VPackValue(getType()));
 }
 
-ResultT<EdgeDefinition> EdgeDefinition::createFromVelocypack(VPackSlice edgeDefinition) {
+ResultT<EdgeDefinition> EdgeDefinition::createFromVelocypack(VPackSlice edgeDefinition, std::set<std::string> const& satCollections) {
```

Comment:
```suggestion
ResultT<EdgeDefinition> EdgeDefinition::createFromVelocypack(VPackSlice edgeDefinition, std::unordered_set<std::string> const& satCollections) {
```

Reply:
There is no need for the collections to be sorted. So we would loose some speed here